svg/util.go

```diff
@@ -0,0 +1,42 @@
+package svg
+
+import (
+	"fmt"
+	"math"
+	"strings"
+
+	"github.com/tdewolff/canvas"
+	"github.com/tdewolff/minify/v2"
+)
+
+// equal returns true if a and b are equal with tolerance Epsilon.
+func equal(a, b float64) bool {
```

Comment:
We could probably export this from `canvas`  as `Equal`.

Reply:
exported!